resources/run_adf.sh

```diff
@@ -77,22 +81,22 @@ MONGOSH_DOWNLOAD_BASE=https://downloads.mongodb.com/compass
 
 ## Linux
 # Ubuntu 22.04
-MONGO_DOWNLOAD_UBUNTU=mongodb-linux-x86_64-ubuntu2204-6.0.4.tgz
+MONGO_DOWNLOAD_UBUNTU=mongodb-linux-x86_64-ubuntu2204-7.0.0-rc6.tgz
```

Comment:
personal question, but wondering the cadence for when we change these. I.e. is it just whenever we see a need (useful features, major release, etc)? and are there any particular tests to run, or more just eyeballing if existing tests work?

Reply:
I created this ticket to address possibly automating updating the versions.  We haven't really discussed the cadence, so far it's been when someone brings it up or makes a major change to run_adf. 
https://jira.mongodb.org/browse/SQL-981